examples/mnist/Instructions.md

```diff
@@ -0,0 +1,30 @@
+<p align="center"><b>Setup, build and run tinyddn on various OS</b><p align="center">
+
+**tiny-dnn** is a C++11 implementation of deep learning. It is suitable for deep learning on limited computational resource, embedded systems and IoT devices.
+
+
+<p align="center"><b>How to Install/Build TinyDnn On Mac/Linux</b></p>
+**Step:1** Install Cmake from https://cmake.org/install/  
+**Step:2** Open Terminal (Press CMD+SpaceBar Enter Terminal)    
+**Step:3** 
+```cpp
```

Comment:
You definitely don't need to specify C++ as a language). Actually you should remove this at all I think